packages/backend-storage/src/construct.ts

```diff
@@ -63,6 +64,13 @@ export type AmplifyStorageProps = {
       ConstructFactory<ResourceProvider<FunctionResources>>
     >
   >;
+  /**
+   * Group the storage resource with existing Amplify resources or separate the storage resource into its own group.
+   *
+   * Warning: Changing this is a destructive change for the storage resource.
```

Comment:
I'm not sure if there is a way to prompt if `resourceGroupName` value changes for storage since this package isn't CLI-like.

I was also thinking if we should add another prop like `allowDestructiveChanges = true` and error if `resourceGroupName` has a value and this is false.

Reply:
We're supporting resourceGroupName for storage too?

Reply:
I don't think we should support this for storage just yet , but if we need it to cover an existing use case then we'll need a test around it

Reply:
Currently there's no use case reported, I'll remove it for now and we can add this back when we need to expand to storage.